sections/dashboard/FuturesMarketsTable/FuturesMarketsTable.tsx

```diff
@@ -436,6 +437,15 @@ const StyledText = styled.div`
 	font-family: ${(props) => props.theme.fonts.bold};
 `;
 
+const ColoredPrice = styled.div<{ color: string }>`
```

Comment:
Should we make this a dedicated component and move to components folder, then we could use it throughout for price display. Also if we wanted to add some subtle effect like fade back to white it would be contained in one component then.